### ISSUE TITLE
[AZ] PCL Cases Historical

### ISSUE THREAD

**Issue by pscsharon** [COVID19Tracking/issues#614]
1. On June 11, CTP began tracking confirmed and probable values separately in AZ. Previously, we captured lumped values. However, Arizona has reported confirmed values separately prior to June 11. We should check to see if we can backfill these values separately.

<img width="1460" alt="Screen Shot 2020-07-13 at 7 18 58 AM" src="https://user-images.githubusercontent.com/67423649/87317551-04c3a000-c4dc-11ea-80b1-cbc747634d6e.png">

**Comment by brianskli** [comment]
Note from 7/20: The epi-curve cannot be used to accurately backfill, as it refers to specimen collection data instead of date of test result report.

**Comment by jesseandersonumd** [comment]
**Update (7/30):** 

- The GH issue says that we can potentially backfill prior to June 11 with the epi curve. Like Brian added, this graphic collects data for **specimens, not people**. Fundamentally, we cannot compare these units and thus _we are unsure whether to backfill unless the units are uniform._
- During the data enhancements shift, we looked for possible statements on AZ's website that would help us decide whether we could backfill. **_We could not find a source that would provide backfill-able values for positive cases (PCR) prior to June 11._** We also did not find any specific answers or resources in outreach's AnswerLog/QuestionLog.
